lib/configs/configs.dart

```diff
@@ -0,0 +1,19 @@
+const clientId = '<client_id>';
```

Comment:
Better to mention types of the variables. If it is a lot of work do it gradually. Not mandatory for this PR.

Reply:
fixed